tests/e2e/scenarios/test_reborn_webui_v2_legacy_approval.py

```
@@ -0,0 +1,541 @@
+        )
+
+    async def handle_threads(route):
+        await fulfill_json(
+            route,
+            {
+                "threads": thread_records,
+                "next_cursor": None,
+            },
+        )
+
+    async def handle_timeline(route):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `handle_timeline`

Add return type annotation: `None`

(ANN202)